src/plugins/pbjx/components/revert-view-modal/RevertDetails.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import isPlainObject from 'lodash/isPlainObject'
+import RevertPropertiesTable from '../revert-properties-table';
+import filterData from '../../utils/filterData';
+import findNodeDiff from '../../utils/findNodeDiff';
+
+/**
+ * This will keep the whole object or list for a key. The reason is trying
+ * to revert specific keys in a block list or multiselect list is difficult
+ * because the data might have changed greatly. So we just put it back
+ * completely to the original state.
+ * 
+ * @param {Object} newNode 
+ * @param {Object} origNode 
+ */
+const fullMapsAndLists = (newNode, origNode) => {
+  for (let [key, value] of Object.entries(newNode)) {
+    if (Array.isArray(value) || isPlainObject(value)) {
+      newNode[key] = origNode[key];
+    }
+  }
+  return newNode;
+}
+
+const RevertDetails = ({ event, isFieldSelected, onSelectField: handleSelectField }) => {
+  const schema = event.schema();
+
+  if (schema.hasMixin('gdbots:ncr:mixin:node-updated')) {
```

Comment:
@rsumilang is it possible for this to be anything else? the event stream component only shows the button when it is an updated event